controllers/create_rolesbindings.go

```diff
@@ -137,6 +137,16 @@ func getRules() []rbacv1.PolicyRule {
 			Resources: []string{"tokenreviews"},
 			Verbs:     []string{"create"},
 		},
+		{
+			APIGroups: []string{"authorization.k8s.io"},
+			Resources: []string{"selfsubjectaccessreviews"},
+			Verbs:     []string{"create"},
+		},
```

Comment:
Please combine into a single object like this.
```suggestion
		{
			APIGroups: []string{"authorization.k8s.io"},
			Resources: []string{"selfsubjectaccessreviews", "selfsubjectrulesreviews"},
			Verbs:     []string{"create"},
		},
```